django/db/backends/oracle/operations.py

```diff
@@ -58,6 +59,16 @@ class DatabaseOperations(BaseDatabaseOperations):
         'TextField': cast_char_field_without_max_length,
     }
 
+    unsupported_distinct_aggregates = (
+        aggregates.StdDev,
```

Comment:
`Variance` is also unsupported as we use `VAR_POP` and `VAR_SAMP` depending on the value of the `sample` argument, neither of which support `DISTINCT` on Oracle according to documentation. This is the same as for the `StdDev` function; and for both of them on MySQL. Perhaps it is worth double-checking the support for all aggregates?

Hmm. Also, should we have had a test failure for `Variance(distinct=True)` on Oracle? I see Tim has highlighted the lack of tests...